src/performance.js

```diff
@@ -120,9 +120,11 @@ export class Performance {
 
     this.whenViewportLayoutComplete_().then(() => {
       if (didStartInPrerender) {
-        const userPerceivedVisualCompletenesssTime = docVisibleTime > -1 ?
-            (timer.now() - docVisibleTime) : 0;
-        this.tick('pc', undefined, userPerceivedVisualCompletenesssTime);
+        const userPerceivedVisualCompletenesssTime = docVisibleTime > -1
+            ? (timer.now() - docVisibleTime)
+            : 1 /* MS (magic number for prerender was complete
```

Comment:
ah nvm, i get it (we already use 0 in tickDelta) and we use 1 as the relative time to it.